### PR TITLE
Documentation re-wroding

### PR DESCRIPTION
**Pull request by pirkka**
Clarified the current caveats section to match my experience.

I did this in the master branch... if you want me to re-create this pull request in a "feature branch" I can do that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,28 +97,26 @@ Set your configuration in the generated `config/sync.yml` file, using the Pusher
...
 
 ```erb
-<table>
-  <tbody>
-    <tr>
-      <%= sync partial: 'todo_row', resource: @todo %>
-    </tr>
-  </tbody>
-</table>
+<div class="todo-items>
```
Comment (ajb) [inline README.md]: missing a quote

File: README.md [hunk 2]
```diff
@@ -97,28 +97,26 @@ Set your configuration in the generated `config/sync.yml` file, using the Pusher
...
+  <%= sync partial: 'todo_row', resource: @todo %>
```
Comment (ajb) [inline README.md]: would be nice to standardize on single/double quotes.

File: README.md [hunk 3]
```diff
@@ -127,20 +125,19 @@ And the parent view:
 sync partial `_todo_row.html.erb`:
 
 ```erb
-<tr> <!-- root level container for the partial required here -->
+<div class="todo-item"> <!-- root level container for the partial added -->
   Title:
   <%= link_to todo.title, todo %>
-</tr>
+  <!-- <br/> probably we don't need this line break anymore :) -->
```
Comment (ajb) [inline README.md]: Funny, but distracts from the main content IMO.

File: README.md [hunk 4]
```diff
@@ -127,20 +125,19 @@ And the parent view:
...
 
 And the parent view changed to:
 
 ```erb
-<table>
-  <tbody>
-    <%= sync partial: 'todo_row', resource: @todo %>
-  </tbody>
-</table>
+<div class="todo-items>
```
Comment (ajb) [inline README.md]: Missing the same quote here.

**Comment by ajb** [comment]
Thanks for the PR! I know you and @chrismccord discussed this in #178, but can you clarify how you think these changes make the documentation clearer? I'm trying to see it, but I really can't see how this is any more friendly.

**Comment by pirkka** [comment]
I guess it's subjective. :)

As for details, I removed the table because I thought it was weird pushing
multiple "todo items" inside a TR tag. Also I was wondering if the <br/>
thing was too much myself, but I thought it would make the whole code
example more realistic.

Pirkka

**Pull request closed without merging**